lumigator/python/mzai/backend/jobs/submission.py

```diff
@@ -12,9 +12,9 @@
 class RayJobEntrypoint(ABC):
     """A generic command which is passed a config and submitted as a ray job.
 
-    Currently the only command we run is `lm-buddy evaluate`, but this can
-    be parametrised to support both different parameters (e.g. `lm-buddy
-    evaluate lm-harness`, `lm-buddy finetune`, etc) or even different commands.
+    Currently the only command we run is `evaluator evaluate`, but this can
```

Comment:
We are about to develop a batch inference job. Will this be "evaluator predict" or do we want to get back to a more general name so we do e.g. "rayjob evaluate" and "rayjob predict"? @veekaybee @chainlink